pkg/tuned/host_payload.go

```diff
@@ -0,0 +1,74 @@
+package tuned
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+
+	"k8s.io/klog"
+	"k8s.io/utils/pointer"
+
+	ign3types "github.com/coreos/ignition/v2/config/v3_1/types"
+)
+
+func stalldIgnitionFile() ign3types.File {
+	const stalldPath = "/usr/local/bin/stalld"
+	mode := 0755
+
+	stalldBytes, err := ioutil.ReadFile(stalldPath)
+	if err != nil {
+		klog.Errorf("failed to read %s: %v", stalldPath, err)
+		return ign3types.File{}
+	}
+	payload := fmt.Sprintf("data:application/octet-stream;base64,%s", base64.StdEncoding.EncodeToString(stalldBytes))
+
+	return ign3types.File{
+		Node:          ign3types.Node{Path: stalldPath},
+		FileEmbedded1: ign3types.FileEmbedded1{Contents: ign3types.Resource{Source: &payload}, Mode: &mode}}
+}
+
+func stalldSystemdUnit() ign3types.Unit {
+	unit := `[Unit]
+Description=Stall Monitor
+
+[Service]
+Environment=CLIST=
+Environment=AGGR=
+Environment=BP="-p 1000000000"
+Environment=BR="-r 20000"
+Environment=BD="-d 3"
```

Comment:
really just a suggestion: long options could be easier to understand, `boost_period` instead of `BP` or `p`

```suggestion
Environment=BP="--boost_period 1000000000"
Environment=BR="--boost_runtime 20000"
Environment=BD="--boost_duration 3"
```

Reply:
Agreed, however, I tried to diverge from the upstream as little as possible (https://github.com/bristot/stalld/blob/master/redhat/stalld.conf).  Honestly not sure which is better.  Diverge or have long options.

Reply:
Added comments in the unit to match the upstream `stalld.conf`.